src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Completion/TagHelperServiceTestBase.cs

```diff
@@ -107,6 +107,7 @@ public TagHelperServiceTestBase()
                     parameter.SetPropertyName("Something");
                 });
             });
+            directiveAttribute1.Metadata[TagHelperMetadata.Common.ClassifyAttributesOnly] = bool.TrueString;
```

Comment:
I had to add these to make `@test` attributes behave correctly now that we're testing against IsAttributeMatch. As far as I can tell they always should have been there.